src/main/java/uk/gov/companieshouse/api/accounts/interceptor/LoggingInterceptor.java

```diff
@@ -0,0 +1,54 @@
+package uk.gov.companieshouse.api.accounts.interceptor;
+
+import static uk.gov.companieshouse.api.accounts.util.AccountsLogUtil.START_TIME_KEY;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.springframework.web.servlet.ModelAndView;
+import org.springframework.web.servlet.handler.HandlerInterceptorAdapter;
+import uk.gov.companieshouse.api.accounts.util.AccountsLogUtil;
+import uk.gov.companieshouse.api.accounts.util.AccountsLogger;
+import uk.gov.companieshouse.api.accounts.util.RequestContext;
+
+@Component
+public class LoggingInterceptor extends HandlerInterceptorAdapter {
+
+    @Autowired
+    private AccountsLogger accountsLogger;
+
+    @Override
+    public boolean preHandle(HttpServletRequest request, HttpServletResponse response,
+        Object handler) {
+        RequestContext requestContext = new RequestContext(requestPath(request),
+            request.getMethod(), requestId(request), userId(request));
```

Comment:
Please can we keep this consistent with the other ways we retrieve info from the request, as below? I.e. rather than calling `request.getMethod()` can we call `requestMethod(request)` and have a new method below which returns the method name. Again, very minor but keeps consistency.